aws_tags.go

```diff
@@ -43,11 +63,30 @@ func createTagSession(region *string, roleArn string) *r.ResourceGroupsTaggingAP
 }
 
 func createASGSession(region *string, roleArn string) autoscalingiface.AutoScalingAPI {
-	sess, err := session.NewSession()
-	if err != nil {
-		panic(err)
+
+	sess := session.Must(session.NewSessionWithOptions(session.Options{
+		SharedConfigState: session.SharedConfigEnable,
+	}))
+
+	maxCloudwatchRetries := 20
+	level := aws.LogDebugWithHTTPBody
+
+	config := &aws.Config{
+		Region: region,
+		HTTPClient: NewHTTPClientWithSettings(HTTPClientSettings{
+			Connect:          55 * time.Second,
+			ExpectContinue:   3 * time.Second,
+			IdleConn:         120 * time.Second,
+			ConnKeepAlive:    30 * time.Second,
+			MaxAllIdleConns:  100,
```

Comment:
Make MaxAllIdleConns = MaxHostIdleConns, there doesn't seem to be a reason to make them different, since there's only one host.